main.go

```diff
@@ -554,10 +554,22 @@ func findAndIterateOverDesiredStates(fileOrDir string, converge func(*state.Helm
 	noTargetFoundForAllHelmfiles := true
 	for _, f := range desiredStateFiles {
 		logger.Debugf("Processing %s", f)
-		yamlBuf, err := tmpl.NewFileRenderer(ioutil.ReadFile, "", environment.EmptyEnvironment).RenderTemplateFileToBuffer(f)
+
+		fileRenderer := tmpl.NewFileRenderer(nil, "", environment.EmptyEnvironment)
```

Comment:
I suspect if the `nil` as the first argument here makes any `readFile` call inside helmfile.yaml fail at first pass.

Reply:
Fixed it in 2072ede